### ISSUE TITLE
unknown protocol: ssh

### ISSUE THREAD

**Issue by coberscheid** [oxygenxml/oxygen-git-plugin#128]
I use Oxygen on a Mac. I had a Git repository already cloned when I installed oxygen-git-plugin. The plugin shows the unstaged and staged files.

But on startup, it asks for my ssh passphrase even though I don't use a passphrase. It also shows "Cannot reach remote repository" with "SSH passphrase required" in the details on hover, which IMO is wrong.

I followed advice from #114 and used log4j.propterties. This is what I found in the log:

```
14749 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitAccess - User home /Users/co
14749 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitAccess - SSH dir exists true
14750 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitAccess - SSH resource path /Users/co/.ssh/id_rsa
14750 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitAccess - SSH resource path /Users/co/.ssh/id_rsa.pub
14750 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitAccess - SSH resource path /Users/co/.ssh/known_hosts
14750 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitAccess - Fire operation successfully ended: WorkinCopyGitEventInfo [Operation: OPEN_WORKING_COPY, working-copy: /Users/co/Work/kwaest.io/doctronic-kwaestio].
14750 DEBUG [ pool-3-thread-1 ] com.oxygenxml.git.service.GitListeners - Fire operation successfully ended: WorkinCopyGitEventInfo [Operation: OPEN_WORKING_COPY, working-copy: /Users/co/Work/kwaest.io/doctronic-kwaestio].
14832 DEBUG [ Thread-16 ] com.oxygenxml.git.service.GitAccess - Begin fetch
14833 DEBUG [ Thread-16 ] com.oxygenxml.git.service.GitAccess - java.net.MalformedURLException: unknown protocol: ssh
java.net.MalformedURLException: unknown protocol: ssh
        at java.net.URL.<init>(URL.java:600) ~[?:1.8.0_202]
        at java.net.URL.<init>(URL.java:490) ~[?:1.8.0_202]
        at java.net.URL.<init>(URL.java:439) ~[?:1.8.0_202]
        at com.oxygenxml.git.service.GitAccess.getHostName(GitAccess.java:1542) ~[oxygen-git-client-2.5.1.jar:?]
        at com.oxygenxml.git.service.GitAccess.fetch(GitAccess.java:1749) ~[oxygen-git-client-2.5.1.jar:?]
        at com.oxygenxml.git.view.ToolbarPanel.fetch(ToolbarPanel.java:269) ~[oxygen-git-client-2.5.1.jar:?]
        at com.oxygenxml.git.view.ToolbarPanel.access$300(ToolbarPanel.java:78) ~[oxygen-git-client-2.5.1.jar:?]
        at com.oxygenxml.git.view.ToolbarPanel$1.lambda$operationSuccessfullyEnded$0(ToolbarPanel.java:243) ~[oxygen-git-client-2.5.1.jar:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_202]
```

The ssh key found is the right one. Is there anything I can do to make this work?

**Comment by dorinajitianu** [comment]
Hi,
Did you update to the latest version, 2.5.1, before this begun to happen? If that's the case, to get the previous version, 2.5.0, you need to:
1. Uninstall version 2.5.1 and restart oxygen
2. [Download version 2.5.0](https://github.com/oxygenxml/oxygen-git-plugin/releases/download/2.5.0/oxygen-git-client-2.5.0-plugin.jar)
3. Unzip it inside {OxygenInstallDir}/plugins . Afterwards, the file system should look like this:
{oxygenInstallDir}/plugins/oxygen-git-client-2.5.0/plugin.xml
4. Restart Oxygen

I currently investigate the and I will release a new plugin version while I understand what's happening.

**Comment by sorincarbunaru** [comment]
A new version has been released (2.5.2) and it should fix the issue. Let us know if somehow it still persists.

**Comment by coberscheid** [comment]
Thx for the quick fix. I have one issue left when pushing, but I assume this to be a local configuration problem with Gerrit. So thx again.
